reference/webhooks.v3.yml

```diff
@@ -1,463 +1,642 @@
 openapi: 3.0.0
 info:
-  title: Webhooks v3
+  title: Webhooks
+  description: |
+    Create and manage webhooks to get notified when specific events occur on stores. For more information, [see Webhooks Overview](https://developer.bigcommerce.com/api-docs/store-management/webhooks/overview).
+
+     <div class="HubBlock--callout">
+     <div class="CalloutBlock--info">
+     <div class="HubBlock-content">
+    <!-- type:info -->
+
+    > ### Note
+    > * Webhooks endpoints are available on both the V2 and V3 REST API.
+
+    </div>
+    </div>
+    </div>
   version: ''
-  description: 'Get notified when specific events occur on a BigCommerce store. For more information, see [Webhooks Overview](https://developer.bigcommerce.com/api-docs/store-management/webhooks/overview).'
+  contact: {}
+servers:
+  - url: 'https://api.bigcommerce.com/stores/{$$.env.store_hash}/v3'
+    variables:
+      $$.env.store_hash:
+        default: '{$$.env.store_hash}'
+  - url: 'https://api.bigcommerce.com/stores/{$$.env.store_hash}/v2'
+    variables:
+      $$.env.store_hash:
+        default: '{$$.env.store_hash}'
 paths:
   /hooks:
-    post:
+    get:
+      operationId: getAllWebhooks
+      summary: Get Webhooks
+      description: |-
+        Returns a list of all webhooks on a store associated to the `client_id` used to authenticate the request.
+
+        <div class="HubBlock--callout">
+        <div class="CalloutBlock--info">
+        <div class="HubBlock-content">
+
+        <!-- type:info -->
+
+        > ### Note
+        > * Webhooks endpoints are available on both the V2 and V3 REST API. 

```

Comment:
Do we need to mention the V2 API in this documentation?

Reply:
I left the mention since it was in the API beta docs. Let's see what @uwikaiddi says about whether we should break this spec file into two menu items for `V2` and `V3` or leave consolidated, in which case we should probably point out that both versions are available.